rules/0015-wazuh_rules.xml

```diff
@@ -1,63 +1,229 @@
 <!--
-  -  OSSEC rules
-  -  Author: Daniel Cid.
-  -  Updated by Wazuh, Inc.
+  -  Wazuh rules
+  -  Created by Wazuh, Inc.
```

Comment:
We should still mention _OSSEC_ in the file header.

Reply:
Maybe now that file name has changed, we should name this like i.e. `Wazuh rules Created by Wazuh, Inc. Based on OSSEC rules, credits to Daniel Cid.`

Reply:
Fixed.